src/components/checkboxes/index.md.njk

```diff
@@ -109,6 +127,11 @@ For example, ‘Select if you are British, Irish or a citizen of a different cou
 Say ‘Select [whatever it is]’.<br>
 For example, ‘Select your nationality or nationalities’.
 
+#### If both a ‘none’ checkbox and another checkbox are checked
```

Comment:
I wonder could we make the heading title active. For example, something like:
```suggestion
#### If users check both a 'none' checkbox' and another checkbox
```

Reply:
👍  (fixed typo of third quotation mark)